Core/scripts/dirac-install.py

```diff
@@ -1,2724 +1,49 @@
-                                 "scripts", "dirac_deploy_scripts.py")
-    if os.path.isfile(ddeLocation):
-      cmd = ddeLocation
-
-      # if specified, create symlink instead of wrapper.
-      if cliParams.scriptSymlink:
-        cmd += ' --symlink'
-
-      # In MacOS /usr/bin/env does not find python in the $PATH, passing binary path
-      # as an argument to the dirac-deploy-scripts
-      if not cliParams.platform:
-        cliParams.platform = getPlatform()
-      if "Darwin" in cliParams.platform:
-        binaryPath = os.path.join(cliParams.targetPath, cliParams.platform)
-        logNOTICE("For MacOS (Darwin) use explicit binary path %s" % binaryPath)
-        cmd += ' %s' % binaryPath
-
-      os.system(cmd)
-    else:
-      logERROR("No dirac-deploy-scripts found. This doesn't look good")
-  else:
-    logNOTICE("Skipping installing DIRAC")
 
-  # we install with DIRACOS from v7rX DIRAC release
-  if cliParams.diracOS \
-     or list(releaseConfig.prjRelCFG['DIRAC'])[0][1] not in '0123456789' \
-     or int(list(releaseConfig.prjRelCFG['DIRAC'])[0][1]) > 6:
-    logNOTICE("Installing DIRAC OS %s..." % cliParams.diracOSVersion)
-    if cliParams.pythonVersion.startswith("3"):
-      if not installDiracOSPython3(releaseConfig):
-        sys.exit(1)
-    else:
-      if not installDiracOS(releaseConfig):
-        sys.exit(1)
-    if not createBashrcForDiracOS():
-      sys.exit(1)
-  else:
-    logNOTICE("Installing %s externals..." % cliParams.externalsType)
-    if not installExternals(releaseConfig):
-      sys.exit(1)
-    if cliParams.noLcg:
-      logNOTICE("Skipping installation of LCG software...")
-    else:
-      logNOTICE("Installing LCG software...")
-      if not installLCGutils(releaseConfig):
-        sys.exit(1)
-    if not createBashrc():
-      sys.exit(1)
-    if not createCshrc():
-      sys.exit(1)
-  if not createOldProLinks():
-    sys.exit(1)
-  runExternalsPostInstall()
-  writeDefaultConfiguration()
-  if cliParams.externalsType == "server":
-    fixMySQLScript()
-  installExternalRequirements(cliParams.externalsType)
-  if cliParams.createLink:
-    createSymbolicLink()
-  logNOTICE("%s properly installed" % cliParams.installation)
-  sys.exit(0)
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("\n")
+sys.stderr.write("Getting dirac-install from this location is no longer supported!\n")
+sys.stderr.write("\n")
+sys.stderr.write("Please update your scripts to use:\n")
+sys.stderr.write("    https://raw.githubusercontent.com/DIRACGrid/management/add-dirac-install/dirac-install.py\n")
+sys.stderr.write("\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+
```

Comment:
Fail if the environment variable DIRAC_DEPRECATED_FAIL is set

Reply:
Done